cmd/mdatagen/loader.go

```diff
@@ -50,15 +50,30 @@ type metric struct {
 	// Unit of the metric.
 	Unit string `yaml:"unit"`
 
-	// Raw data that is used to set Data interface below.
-	YmlData *ymlMetricData `yaml:"data" validate:"required"`
-	// Date is set to generic metric data interface after validating.
-	Data MetricData `yaml:"-"`
+	// Sum stores metadata for sum metric type
+	Sum *sum `yaml:"sum"`
+	// Gauge stores metadata for gauge metric type
+	Gauge *gauge `yaml:"gauge"`
+	// Histogram stores metadata for histogram metric type
+	Histogram *histogram `yaml:"histogram"`
```

Comment:
What do you think between these 2:

```yaml
metrics:
  system.cpu.time:
    description: Total CPU seconds broken down by different states.
    unit: s
    data:
      sum:
        monotonic: true
        aggregation: cumulative
    attributes:
```

VS 

```yaml
metrics:
  system.cpu.time:
    description: Total CPU seconds broken down by different states.
    unit: s
    sum:
      monotonic: true
      aggregation: cumulative
    attributes:
```

Reply:
The first one maybe more semantically correct, but I would lean towards better user experience here: 2.